documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -480,3 +547,52 @@ Default: `1`
 After reading `config.txt` the GPU firmware `start4.elf` reads the bootloader EEPROM config and checks for a section called `[config.txt]`. If the `[config.txt]` section exists then the contents from the start of this section to the end of the file is appended in memory, to the contents of the `config.txt` file read from the boot partition.  This can be used to automatically apply settings to every operating system, for example, dtoverlays.
 
 WARNING: If an invalid configuration which causes boot to fail is specified then the bootloader EEPROM will have to be re-flashed.
+
+[[bootloader_update_stable]]
+=== Updating to the LATEST / STABLE bootloader
+
+The DEFAULT version of the bootloader is only updated for CRITICAL fixes and major releases. The LATEST / STABLE bootloader is updated more often to include the latest fixes and improvements.
+
+Advanced users can switch to the LATEST / STABLE bootloader to get the latest functionality.
+Open a command prompt and start `rasp-config`.
+
+----
+sudo raspi-config
+----
+
+Navigate to `Advanced Options` and then `Bootloader Version`. Select `Latest` and choose `Yes` to confirm. Select `Finish` and confirm you want to reboot. After the reboot, open a command prompt again and update your system.
```

Comment:
Hmm, if the user did the `sudo apt update; sudo apt upgrade` **before** running `raspi-config`, would that reduce the number of reboots required?

Reply:
Probably - they won't see an update is available unless they ignore the reboot request in raspi-config - but I guess that's ok.

Reply:
Actually I'm wrong. You have to update rpi-eeprom, but that will have no effect until you've ran raspi-config to switch the bootloader version. I don't really want to advise them to ignore the reboot request in raspi-config even though it's unlikely to be an issue in this case.

Reply:
Left this alone - advanced users will probably know what they're doing?

Reply:
I wasn't suggesting that they should _ignore_ the raspi-config reboot, but AIUI the instructions currently say:
1. run raspi-config to switch to 'latest' bootloader stream
2. reboot
3. update rpi-eeprom apt package
4. run rpi-eeprom-update to actually update the eeprom
5. reboot

I was wondering if this sequence would work instead:
1. update rpi-eeprom apt package
2. run raspi-config to switch to 'latest' bootloader stream
3. reboot

But I won't labour the point, as you're obviously documenting "we know this works" procedure :slightly_smiling_face: 
(and there's more than a good chance that I'm misunderstanding things)